apps/condo/domains/resident/schema/RegisterServiceConsumerService.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Generated by `createservice resident.RegisterServiceConsumerService --type mutations`
+ */
+
+const { getById, GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/resident/access/RegisterServiceConsumerService')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const { BillingIntegrationOrganizationContext, BillingAccount } = require('@condo/domains/billing/utils/serverSchema')
+const { ServiceConsumer, Resident } = require('../utils/serverSchema')
+const { NOT_FOUND_ERROR } = require('@condo/domains/common/constants/errors')
+
+
+const RegisterServiceConsumerService = new GQLCustomSchema('RegisterServiceConsumerService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterConsumerServiceInput { dv: Int!, sender: JSON!, residentId: String!, unitName: String!, accountNumber: String! }',
```

Comment:
`residentId: String`  we have some special type for relations

Reply:
Replaced to ID!